dos/src/main.rs

```diff
@@ -1113,14 +1127,24 @@ pub mod test {
             .unwrap();
         let nodes_slice = [node];
 
-        let client = Arc::new(ThinClient::new(
-            cluster.entry_point_info.rpc().unwrap(),
-            cluster
-                .entry_point_info
-                .tpu(cluster.connection_cache.protocol())
-                .unwrap(),
-            cluster.connection_cache.clone(),
-        ));
+        let rpc_pubsub_url = format!("ws://{}/", cluster.entry_point_info.rpc_pubsub().unwrap());
```

Comment:
myabe move this common code to create TpuClient to some helper function in the scope of tests submodule?

Reply:
good idea. will do